saleor/product/models.py

```diff
@@ -512,9 +512,9 @@ def display_product(self, translated: bool = False) -> str:
         )
         return smart_text(product_display)
 
-    def get_first_image(self) -> "ProductImage":
-        images = list(self.images.all())
-        return images[0] if images else self.product.get_first_image()
+    def get_first_image(self) -> "ProductMedia":
+        media = list(self.media.all())
```

Comment:
Just to double-check. Are we sure that this media will be always an image?

Reply:
Nope, needs to be changed